.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: haskell
+
+ghc:
```

Comment:
Deploy what, Carnival? This is a general purpose library, I think we should support as much as we can, to a reasonable limit. I think latest and 1 or 2 back is reasonable, assuming supporting 7.6 isn't a huge jungle of CPP. Since the decision's not been thought about yet, I think it makes sense to see what it currently works with now and support that until something prompts us to think about it again.